database-commons/src/main/java/io/cdap/plugin/util/DBUtils.java

```diff
@@ -59,7 +59,7 @@
 public final class DBUtils {
   private static final Logger LOG = LoggerFactory.getLogger(DBUtils.class);
 
-  private static final Calendar PURE_GREGORIAN_CALENDAR = createPureGregorianCalender();
+  public static final Calendar PURE_GREGORIAN_CALENDAR = createPureGregorianCalender();
```

Comment:
Why this change ?

Reply:
this was getting used in one of the class and was public in 1.11 release

Reply:
If it is not getting used in any of the redshift class related to your change, let's revert. Otherwise LGTM.

Reply:
It is getting used.